language/compiler/ir-to-bytecode/src/compiler.rs

```diff
@@ -1362,3 +1409,196 @@ fn compile_call(
         }
     })
 }
+
+//**************************************************************************************************
+// Bytecode
+//**************************************************************************************************
+
+fn compile_function_body_bytecode(
+    context: &mut Context,
+    formals: Vec<(Var, Type)>,
+    locals: Vec<(Var, Type)>,
+    blocks: BytecodeBlocks,
+) -> Result<CodeUnit> {
+    let mut function_frame = FunctionFrame::new();
+    let mut locals_signature = LocalsSignature(vec![]);
+    for (var, t) in formals {
+        let sig = compile_type(context, &t)?;
+        function_frame.define_local(&var.value, sig.clone())?;
+        locals_signature.0.push(sig);
+        record_src_loc!(local: context, var);
+    }
+    for (var_, t) in locals {
+        let sig = compile_type(context, &t)?;
+        function_frame.define_local(&var_.value, sig.clone())?;
+        locals_signature.0.push(sig);
+        record_src_loc!(local: context, var_);
+    }
+    let sig_idx = context.locals_signature_index(locals_signature)?;
+
+    let mut code = vec![];
+    let mut label_to_index: HashMap<Label, u16> = HashMap::new();
+    for (label, block) in blocks {
+        label_to_index.insert(label.clone(), code.len() as u16);
+        context.label_index(label)?;
+        compile_bytecode_block(context, &mut function_frame, &mut code, block)?;
+    }
+    let fake_to_actual = context.build_index_remapping(label_to_index);
+    remap_branch_offsets(&mut code, &fake_to_actual);
+    let max_stack_size = u16::max_value();
```

Comment:
Just to note that max_stack_size is ignored by the VM and will be removed soon. u16::max_value() is fine, but so is any other value.

Reply:
I know, just wanted something that was "technically" valid before it was